keychain_test.go

```diff
@@ -23,7 +21,7 @@ func TestFlow(t *testing.T) {
 		t.Fatal(err)
 	}
 	pass, err = Find(app, "testuser")
-	if err == nil || pass != "" {
+	if err == nil || err.Error() != "The specified item could not be found in the keychain." || pass != "" {
```

Comment:
I'd rather check for `!empty string || "Unknown error"` in case the macOS error message changes (are they localized?)